### PR TITLE
Add themeable color for comment glyphs on lines with unresolved threads

### PR DESCRIPTION
**Pull request by hermannloose**
As a follow-up to #174418 which decorated lines with unresolved comment threads with the `comment-unresolved` icon, we would like the option to distinguish by color as well. Our users request this especially in cases where the comment is collapsed, to quickly spot lines with actionable comments.

### CODE REVIEW

File: src/vs/workbench/contrib/comments/browser/commentGlyphWidget.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@ import { IEditorDecorationsCollection } from 'vs/editor/common/editorCommon';
...
-registerColor('editorGutter.commentGlyphForground', { dark: editorForeground, light: editorForeground, hcDark: Color.black, hcLight: Color.white }, nls.localize('editorGutterCommentGlyphForeground', 'Editor gutter decoration color for commenting glyphs.'));
+export const editorGutterCommentGlyphForeground = registerColor('editorGutter.commentGlyphForground', { dark: editorForeground, light: editorForeground, hcDark: Color.black, hcLight: Color.white }, nls.localize('editorGutterCommentGlyphForeground', 'Editor gutter decoration color for commenting glyphs.'));
+export const editorGutterCommentGlyphForegroundUnresolved = registerColor('editorGutter.commentGlyphForegroundUnresolved', { dark: editorGutterCommentGlyphForeground, light: editorGutterCommentGlyphForeground, hcDark: editorGutterCommentGlyphForeground, hcLight: editorGutterCommentGlyphForeground }, nls.localize('editorGutterCommentGlyphForegroundUnresolved', 'Editor gutter decoration color for commenting glyphs for unresolved comment threads.'));
```
Comment (gjsjohnmurray) [inline src/vs/workbench/contrib/comments/browser/commentGlyphWidget.ts:18]: I think editorGutterCommentUnresolvedGlyphForeground would be a better choice than editorGutterCommentGlyphForegroundUnresolved

**Comment by hermannloose** [comment]
Friendly ping.

**Comment by alexr00** [comment]
@hermannloose I will try to look this week!

**Comment by hermannloose** [comment]
Another friendly ping. 😊

I was also wondering whether it's generally preferred to use merge instead of rebase + force-push for keeping a PR up to date; I noticed that the comment from John earlier resulted in a broken link for a while (but I cannot reproduce that now) after I had addressed the changes and force-pushed.

**Review by alexr00 — APPROVED** [review]
Thank you for your patience with my slowness in reviewing this change! I've fixed the nits so that we can merge this sooner.

**Review by rebornix — APPROVED** [review]

**Pull request merged**